test/suite/test_timestamp03.py

```diff
@@ -269,6 +269,24 @@ def test_timestamp03(self):
         self.check(self.session, 'read_timestamp=' + old_ts,
             self.table_nots_nolog, dict((k, self.value2) for k in orig_keys))
 
+        # Scenario: 4a
+        # This scenario is same as earlier one with read_timestamp earlier than
+         # oldest_timestamp and using the option of round_to_oldest
+        earlier_ts = timestamp_str(90)
+        self.check(self.session,
```

Comment:
We should also test the case where round_to_oldest = false and read_timestamp is older than oldest timestamp.

Reply:
read timestamp older than the oldest timestamp with default (false) round_to_oldest is already tested in test_timestamp09